packages/datadog-core/test/storage/test.js

```diff
@@ -119,6 +124,16 @@ module.exports = factory => {
         done()
       })
     })
+
+    it('should not log ddResourceStore contents', done => {
+      storage.run(store, () => {
+        setImmediate(() => {
+          expect(inspect(executionAsyncResource(), { depth: null }))
```

Comment:
This only checks that we don't use a symbol, so for example if we would store a different property it wouldn't catch a regression. Any way to validate instead that the result is the same with and without dd-trace?

Reply:
Maybe, we can check if we have any property in `Object.getOwnPropertySymbols` and `Object.getOwnPropertyNames`

Reply:
It would have other property names and property symbols, so that wouldn't work. At the least, the Node.js core `owner` symbol will always be there, and most any resource will have a few other internals bits attached. I don't think there are any handle types _without_ extra stuff on them.

Maybe possible to do the with and without comparison? I'll see what I can come up with...

Reply:
They wouldn't have the same value, but they would still have the same keys. It's enough to check that we're not adding keys.

Reply:
We'd still have to parse it though because what we get back is a string, and it's not valid JSON either, so we'd have to do some custom parsing. Not sure it'd be much better than what I already have here.

Reply:
I wrote a thing to parse out the keys. Does this seem like a reasonable enough approach?